release/nightly_tests/dataset/image_loader_microbenchmark.py

```diff
@@ -74,7 +103,10 @@ def tf_crop_and_flip(image_buffer, num_channels=3):
     # allowed range of aspect ratios, sizes and overlap with the human-annotated
     # bounding box. If no box is supplied, then we assume the bounding box is
     # the entire image.
-    shape = tf.shape(image_buffer)[1:]
+    shape = tf.shape(image_buffer)
+    if len(shape) == 4:
```

Comment:
should this be dependent on `num_channels`?
actually, it looks like `num_channels` isn't used in this fn, should we remove it?